extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -51,6 +55,26 @@
   mask-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="image" fill="#fff"><path d="M5,3 L19,3 C20.1045695,3 21,3.8954305 21,5 L21,19 C21,20.1045695 20.1045695,21 19,21 L5,21 C3.8954305,21 3,20.1045695 3,19 L3,5 C3,3.8954305 3.8954305,3 5,3 Z M19,15 L19,5 L5,5 L5,13.0000896 L8,10.0003584 L13,15 L16,12 L19,15 Z M15.5,10 C14.6715729,10 14,9.32842712 14,8.5 C14,7.67157288 14.6715729,7 15.5,7 C16.3284271,7 17,7.67157288 17,8.5 C17,9.32842712 16.3284271,10 15.5,10 Z" id="Combined-Shape"></path></g></g></svg>');
 }
 
+.i-amphtml-lbg-button-next {
+  top: calc(50vh - 18px) !important;
+  right: 0 !important;
+  filter: drop-shadow(0 0 1px black);
+}
+
+.i-amphtml-lbg-button-next .i-amphtml-lbg-icon {
+  mask-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs></defs><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="arrow_right_03" fill="#000000"><path d="M7.75,9.75 L7.75,15.25 C7.75,15.8022847 7.30228475,16.25 6.75,16.25 C6.19771525,16.25 5.75,15.8022847 5.75,15.25 L5.75,8.75 C5.75,8.19771525 6.19771525,7.75 6.75,7.75 L13.25,7.75 C13.8022847,7.75 14.25,8.19771525 14.25,8.75 C14.25,9.30228475 13.8022847,9.75 13.25,9.75 L7.75,9.75 Z" id="Combined-Shape" transform="translate(10.000000, 12.000000) rotate(-225.000000) translate(-10.000000, -12.000000) "></path></g></g></svg>');
+}
+
+.i-amphtml-lbg-button-prev {
```

Comment:
I see some properties in common between `.i-amphtml-lbg-button-next` and `.i-amphtml-lbg-button-prev` that you could probably consolidate into a rule with both of them. e.g.

```css
.i-amphtml-lbg-button-next,
.i-amphtml-lbg-button-prev {
  top: 0;
  bottom: 0;
  height: 36px;
  filter: drop-shadow(0 0 1px black);
}
```

Reply:
Fixed this in latest commit.